formats/json/commonMain/src/kotlinx/serialization/json/JsonElement.kt

```diff
@@ -76,13 +80,52 @@ public fun JsonPrimitive(value: String?): JsonPrimitive {
 @Suppress("FunctionName", "UNUSED_PARAMETER") // allows to call `JsonPrimitive(null)`
 public fun JsonPrimitive(value: Nothing?): JsonNull = JsonNull
 
+/**
+ * Create a [JsonPrimitive] from the given string, without surrounding it in quotes.
+ *
+ * **Note:** Creating a literal unquoted value of `null` (as in, `value == "null"`) is forbidden. If you want to create
```

Comment:
I'd move this note (and without the word 'Note') down, under 'Be aware that'... because it would likely be a rare event